backend/ttrs/views.py

```diff
@@ -150,3 +150,13 @@ class MajorDetail(generics.RetrieveAPIView):
     queryset = Major.objects.all()
     serializer_class = MajorSerializer
     permission_classes = (AllowAny,)
+
+from .recommend import recommend
```

Comment:
It will be good to put this line to the top of this file, although this code is related only `RecommendView` because all other imports are placed to the top of the file.

Reply:
Moved corresponding line to the top of the code.